### PR TITLE
Delete GetTensorInfoCall and relative registry 

### PR DESCRIPTION
**Pull request by qqsun8819**
For now we only have one GetTensorInfo function , so GetTensorInfoCall and relative tensor info registry is no longer needed

### CODE REVIEW

**Comment by kostmo** [comment]
## :pill: CircleCI build failures summary and remediations
As of commit faa3a576:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=faa3a57671e54460927523fce62bf3fd4eb5e56b).

### :detective: 1 new failure recognized by patterns
The following build failures don't appear to be due to upstream breakage:
#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4048531) pytorch_linux_xenial_py3_6_gcc5_4_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=78547691) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=78547691))
<details>
<summary>
<code>Automatic merge failed; fix conflicts and then commit the result.</code>
</summary>

```
CONFLICT (add/add): Merge conflict in .circleci/verbatim-sources/docker_jobs.yml 
Auto-merging .circleci/verbatim-sources/docker_jobs.yml 
CONFLICT (add/add): Merge conflict in .circleci/scripts/python_doc_push_script.sh 
Auto-merging .circleci/scripts/python_doc_push_script.sh 
CONFLICT (add/add): Merge conflict in .circleci/ecr_gc_docker/gc.py 
Auto-merging .circleci/ecr_gc_docker/gc.py 
CONFLICT (add/add): Merge conflict in .circleci/docker/common/install_conda.sh 
Auto-merging .circleci/docker/common/install_conda.sh 
CONFLICT (add/add): Merge conflict in .circleci/config.yml 
Auto-merging .circleci/config.yml 
Automatic merge failed; fix conflicts and then commit the result. 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

**Comment by qqsun8819** [comment]
@jerryzh168 may be take a look at it ^_^

File: caffe2/core/tensor.cc [hunk 1]
```diff
@@ -105,25 +105,6 @@ GetInt8TensorInfo(const void* c, size_t* capacity, DeviceOption* device) {
   return GetTensorInfo(&(int8_tensor->t), capacity, device);
 }
 
-// since we only have one tensor, probably need to remove this at some point?
-static CaffeMap<TypeIdentifier, TensorInfoCall> tensor_info_call_registry_{
-    {TypeMeta::Id<Tensor>(), GetTensorInfo},
-    {TypeMeta::Id<int8::Int8TensorCPU>(), GetInt8TensorInfo},
```
Comment (jerryzh168) [inline caffe2/core/tensor.cc]: looks like we still have `int8::Int8TensorCPU` before we can get rid of this. I believe caffe2 int8 tensor is still in use in production. cc @dskhudia do you know if this is true?

**Comment by qqsun8819** [comment]
any progress for this ? ^_^ @jerryzh168 @ezyang

**Pull request closed without merging**